README.md

```diff
@@ -218,19 +218,45 @@ When running OpenShift in a single VM, it is recommended to reuse the Docker dae
 for pure Docker use-cases as well.
 By using the same docker daemon as Minishift, you can speed up your local experiments.
 
-To be able to work with the docker daemon on your Mac or GNU/Linux host use the
-[`docker-env command`](https://minishift.io/docs/minishift_docker-env.html) in your shell:
+1. Make sure that you have `docker` and `docker-client` installed on your machine.
```

Comment:
@thatdocslady As @hferentschik mentioned, OS X and Windows is our primary target for this usecase. Linux is secondary as User can run native docker. I think we can refer users to https://docs.docker.com/engine/installation/binaries/ for it. For Linux we just need to make sure docker is installed. If we want to use docker client we do not need to start the docker daemon. I think we do not need to put all the details but enough that user can figure it out.